tests/unit/lms/views/basic_lti_launch_test.py

```diff
@@ -386,7 +386,6 @@ def test_it_sets_the_right_javascript_config_settings(
 
         assert context.js_config == {
             "mode": "content-item-selection",
-            "debug": Any.dict(),
```

Comment:
Am I being daft. Why has this gone? Because it's not this guys job anymore?

Reply:
Yep, not this guy's job anymore. And `context.js_config` in these tests isn't the real `JSConfig` it's a fake that contains only the minimum stuff that `BasicLTILaunchViews` cares about:

https://github.com/hypothesis/lms/blob/b9d0c2712bfd7efa04ffde6e8b8d59aa66a08994/tests/unit/lms/views/basic_lti_launch_test.py#L455

This of course is a fake that's duplicating part of the real `JSConfig` and could get out of sync with it. `mock.create_autospec()` will take care of that for us soon, once `JSConfig` no longer exposes a dict directly